spec/lib/twingly/http_spec.rb

```diff
@@ -3,13 +3,13 @@
 class CustomError < StandardError; end
 
 RSpec.describe Twingly::HTTP::Client do
-  let(:logger)            { NullLogger.new }
+  let(:logger)            { Logger.new(File::NULL) }
```

Comment:
An alternative is to not define any logger at all here, and instead define a new client where a logger is needed. 

Decided to keep it like this though, to change as little as possible. Also, if the logger is left out here, the added spec doesn't make sense anymore.

Reply:
I think this is fine too